double/double.mbt

```diff
@@ -12,17 +12,17 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-let not_a_number : Double = 0x7FF8000000000001L.reinterpret_as_double()
+pub let not_a_number : Double = 0.0 / 0.0
 
-let positive_infinity : Double = 0x7FF0000000000000L.reinterpret_as_double()
+pub let infinity : Double = 1.0 / 0.0
 
-let negative_infinity : Double = 0xFFF0000000000000L.reinterpret_as_double()
+pub let neg_infinity : Double = -1.0 / 0.0
 
-let max_val : Double = 0x7FEFFFFFFFFFFFFFL.reinterpret_as_double()
+pub let max : Double = 1.7976931348623157e+308
```

Comment:
I think that using literal to represent floating point number is not precise enough. 

And rust doc and MS doc disagree on the precision `1.7976931348623157E+308f64`[(rust)](https://doc.rust-lang.org/std/primitive.f64.html#associatedconstant.MIN_POSITIVE) vs 
`1.7976931348623158e+308`[(Microsoft)](https://learn.microsoft.com/en-us/cpp/cpp/floating-limits?view=msvc-170)

The old approach `reinterpret_as_double()` is better IMO